kiwi/builder/disk.py

```diff
@@ -703,6 +710,9 @@ def _map_root_filesystem(
         self.storage_map['system'] = filesystem
 
     def _process_build(self, device_map: Dict, disk: Disk) -> None:
+        # create system layout for root system
+        self._create_system_instance(device_map)
```

Comment:
Why is this suddenly necessary?

Reply:
It was there before. The call moved into the function as it imho makes more sense to have it there

Reply:
see the red lines from the diff, you will see the call as an extra line but it only makes sense in combination with the build of the disk

Reply:
Ah, sorry, I completely missed it :facepalm: